css/types/attr.json

```diff
@@ -186,9 +186,9 @@
               }
             }
           },
-          "frequency": {
+          "custom-ident": {
             "__compat": {
-              "description": "&lt;frequency&gt;",
```

Comment:
I can see the `frequency` unit no longer is valid/defined in the spec. My assumption is that it doesn't still need to be recorded? i.e that it hasn't been in any browser for more than two years?